Gemfile.lock

```diff
@@ -166,6 +184,7 @@ DEPENDENCIES
   listen (~> 3.0.5)
   puma (~> 3.0)
   rails (~> 5.0.2)
+  rspec-rails
```

Comment:
why your test in #1 was not written in rspec, but then install rspec as a test engine to trigger for CI 🤔

Reply:
#1 tests were auto generated by `rails generate scaffold`, I did not actually write any test there

Reply:
that means you should config your app to generate rspec style, and regenerate it